### DISCUSSION TITLE
How to revise the specific file to avoid installation when excute make instruction?

### DISCUSSION THREAD

**Discussion by jiejune** [ImageMagick/ImageMagick#2567] [Help]
I am intending to encapsulate some API in  Magick++ for image compression purpose, so i write the following CMakeList:

ExternalProject_Add(ImageMagic
  SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/ImageMagick-7.0.10-29"
  BUILD_IN_SOURCE 1
  CONFIGURE_COMMAND chmod +x ./configure && ./configure
  BUILD_COMMAND $(MAKE)
)

I just want to compile the project without installation. When running the instructions above, some remove and install operations are performed on my equipment. I wonder how to revise the specific file to avoid the installation?

**Comment by urban-warrior** [comment]
Unfortunately we cannot help.  ImageMagick supports automake/autoconf and the developers are unfamiliar with cmake.
